### ISSUE TITLE
simCLR Loss

### ISSUE THREAD

**Issue by pazadimo** [vkola-lab/tmi2022#5]
Hi,

I tried training your simCLR model on my own dataset. The loss started from 6.9 and decreased to 5.02 until the end of epoch 20. I wonder if I should train it for longer epochs or if your loss pattern for your data had a similar pattern.

Thanks a lot.

**Comment by GSWS** [comment]
We used 120 epochs to train the simCLR model. You could stop the training once the loss stops significantly change. The recommended batch size is at least 512. 

Since the simCLR authors mentioned, "when the number of training epochs is small (e.g. 100 epochs), larger batch sizes have a significant advantage over the smaller ones. With more training steps/epochs, the gaps between different batch sizes decrease or disappear..." For better convergence, you need more epochs. 

For more details about training the simCLR, you could visit the official simCLR [repo](https://github.com/google-research/simclr).
